docs/api/browser-window.md

```diff
@@ -1739,14 +1739,14 @@ removed in future Electron releases.
 
 #### `win.setBrowserView(browserView)` _Experimental_
 
-* `browserView` [BrowserView](browser-view.md) | null - Attach browserView to win.
-If there is some other browserViews was attached they will be removed from
+* `browserView` [BrowserView](browser-view.md) | null - Attach `browserView` to `win`.
+If there are other `BrowserView`s attached, they will be removed from
```

Comment:
Aside from the wording changes, I have a question too. Could there be something wrong with markdown rendering over at `electronjs.org`? 

I ask because type of the `browserView` argument here is supposedly `BrowserView | null`. And indeed [on GitHub](https://github.com/electron/electron/blob/master/docs/api/browser-window.md) I see this:
<img width="1012" alt="ongithub" src="https://user-images.githubusercontent.com/3772093/64069320-5765d780-cbfc-11e9-8f29-4d89baa90a69.png">

But over at [the docs given at electronjs.org](https://electronjs.org/docs/api/browser-window#winsetbrowserviewbrowserview-experimental) I see:
<img width="1073" alt="onsite" src="https://user-images.githubusercontent.com/3772093/64069325-6b113e00-cbfc-11e9-86ce-c10238bd4625.png">

The difference doesn't seem trivial; whether `null` is a valid input or not. Should the argument and return types be in backticks?

Reply:
@koralcem that's because the documentation matches the latest **stable** version of Electron, and the `null` argument was added [recently](https://github.com/electron/electron/commit/9910507bc4536342cc3a851675d633dff097b827). It's not a rendering issue :)

Reply:
Ah, thanks. I learned something new today 😄